src/Bugsnag.js

```diff
@@ -269,7 +293,10 @@ export class Report {
       metadata: typedMap(this.metadata),
       severity: this.severity,
       stacktrace: this.stacktrace,
-      user: this.user
+      user: this.user,
+      defaultSeverity: this._eventHandledState.defaultSeverity === this.severity,
```

Comment:
what sets defaultSeverity on _eventHandledState? Should this be originalSeverity?